run_tests.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import signal
+import sys
+import tarfile
+import time
+import urllib2
+
+from subprocess import check_call, Popen, check_output
+
+DIR_TEST_ENV = "test/env"
+DIR_JENKINS_ENV = 'jenkins-env'
+VERSION_VIRTUALENV = '1.9.1'
+
+os.environ["JENKINS_HOME"] = "jenkins-master"
+
+
+def python(*args):
+    pypath = [os.path.join(DIR_TEST_ENV, 'bin', 'python')]
+    check_call(pypath + list(args))
+
+
+def pip(*args):
+    pippath = [os.path.join(DIR_TEST_ENV, 'bin', 'pip')]
```

Comment:
When we have activated the environment ´pip´ should be in the path and you won't need the full path. Constructing that one would make this code fail on Windows, because there is no ´bin´ but a `Scripts` folder.